### DISCUSSION TITLE
ATI Rage Video Card

### DISCUSSION THREAD

**Discussion by WhiteMagicRaven** [86Box/86Box#1787] [Ideas]
https://github.com/dingusdev/dingusppc/blob/master/devices/video/atirage.cpp

This is implementation of ATI Rage card (emulation) i don't know a what status of implementation, but i think and hope it can be helpful to this project.
Thanks for you hard work)

**Comment by OBattler** [comment]
Converting this to a discussion since it's more appropriate for that.

**Reply by anabate123** [reply]
> https://github.com/dingusdev/dingusppc/blob/master/devices/video/atirage.cpp
> 
> This is implementation of ATI Rage card (emulation) i don't know a what status of implementation, but i think and hope it can be helpful to this project. Thanks for you hard work)

I've got the documentation for the ATI Rage Pro card right here...
http://www.vgamuseum.info/index.php/cpu/item/download/117_4a9462201fd34a467ec5797626a88525
http://www.vgamuseum.info/index.php/cpu/item/download/118_15798fb50539a5a89c7a14598f82440b
http://www.vgamuseum.info/index.php/cpu/item/download/119_73c83761c20f30cd62a2a619aaa7b308
http://www.vgamuseum.info/index.php/cpu/item/download/120_6a66704d99c802f4de8eac53a8a3b574
http://www.vgamuseum.info/index.php/cpu/item/download/121_f2e98372d6898857516aa7f460118fdc
